docs/rest-api/Authentication.md

```diff
@@ -27,7 +27,12 @@ const createEvent = async () => {
           values: ["eggs", "bacon", "pasta", "bread"],
         }),
       },
-      { headers: { "x-api-key": "YOUR_API_KEY" } }
+      {
+        headers: {
+          "x-api-key": "YOUR_API_KEY",
+          "Content-Type": "application/json",
+        },
+      }
```

Comment:
Is this not set by default when you send a json body? 

<img width="716" alt="image" src="https://user-images.githubusercontent.com/22891173/207577544-7d460228-2559-41c5-997c-0bc4fdca59b7.png">


<img width="817" alt="image" src="https://user-images.githubusercontent.com/22891173/207577423-40b8bf9c-85a6-4f50-8c8d-d601e392774e.png">


I looked at the discord thread and I guess it depends on the HTTP client you are using, hence my preference for axios here. Perhaps a comment / section under headers in the REST api section - thoughts?

<img width="557" alt="image" src="https://user-images.githubusercontent.com/22891173/207579222-d7f93ae2-f3d3-4445-8e4d-b40edaeb12c4.png">

Reply:
Yea it totally depends on the client and it won't hurt by adding it on but it will totally fail if the client doesn't include it.

We can add a section for headers, but it seems slightly overkill vs just a line in the example. Im good with either though.